packages/lumx-react/src/components/slideshow/Slideshow.tsx

```diff
@@ -182,20 +182,69 @@ export const Slideshow: Comp<SlideshowProps, HTMLDivElement> = forwardRef((props
         onChange(currentIndex);
     }, [currentIndex, onChange]);
 
+    const slideshowSlidesId = 'slideshow-slides';
+    const slideshowId = 'slideshow';
+
+    useEffect(() => {
+        const slideshow = document.getElementById('slideshow');
```

Comment:
We can use the `element` state directly, no need to get the element.

```suggestion
        if (!element) return
        const slideshow = element;
```